OpenRA.Mods.Common/Traits/World/PathFinder.cs

```diff
@@ -33,7 +33,7 @@ public interface IPathFinder
 		/// Calculates a path for the actor from source to destination
 		/// </summary>
 		/// <returns>A path from start to target</returns>
-		List<CPos> FindUnitPath(CPos source, CPos target, Actor self);
+		List<CPos> FindUnitPath(CPos source, CPos target, Actor self, Actor ignoredActor);
```

Comment:
Done. I wasn't sure about - There were few cases outside this class/PR with ignoredActor which I renamed also. There were 3 cases where not parameter but member name was IgnoredActor - renamed also. I kept WithIgnoredActor function name.